blogs/converter.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const marked = require('marked');
+
+var listOfDirectoryFiles = fs.readdirSync('markdown-blogs');
+
+console.log(listOfDirectoryFiles);
+
+var listOfMarkdownFiles = [];
+//find the markdown files and put them in their own array
+for(let i = 0; i < listOfDirectoryFiles.length; i++){
+   const currentFileName = listOfDirectoryFiles[i];
+    const isMarkdown = currentFileName.indexOf(".md");
+    if(isMarkdown != -1){
+
+        listOfMarkdownFiles.push(currentFileName);
+    }
+}
+
+console.log(listOfMarkdownFiles);
+
+for(let i =0; i< listOfMarkdownFiles.length; i++){
```

Comment:
Again here you could use the array type and a for each
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
Done